internal/versions/api.go

```diff
@@ -112,10 +113,30 @@ func getLatestReleaseImageForMajorMinor(releaseImages models.ReleaseImages, majo
 		}
 	}
 
+	log.Debugf("Found latest release image version '%s' for majorMinorVersion '%s'", *latestReleaseImage.Version, majorMinorVersion)
+
 	return latestReleaseImage, nil
 }
 
-func filterReleaseImagesByOnlyLatest(releaseImages models.ReleaseImages, onlyLatest *bool) (models.ReleaseImages, error) {
+// getMultiArchReleaseImage retrieves multi arch release image matching a given x.y.z version.
+func getMultiArchReleaseImage(releaseImages models.ReleaseImages, version string, log logrus.FieldLogger) *models.ReleaseImage {
+	if !strings.HasSuffix(version, "-multi") {
+		version = fmt.Sprintf("%s-multi", version)
+	}
+
+	result := funk.Find(releaseImages, func(releaseImage *models.ReleaseImage) bool {
```

Comment:
nit: we have now introduced [lo](https://github.com/samber/lo), which could replace [funk](https://github.com/thoas/go-funk).
It would improve both type safety (as it's using generics) and readability (we will avoid type assertion making the code more concise).
I would suggest replacing funk for lo functions whenever possible